src/thunderbird_accounts/settings.py

```diff
@@ -333,6 +333,19 @@ def oidc_logout(request):
     # ideally you should turn this off when you're done.
     OIDC_FALLBACK_MATCH_BY_EMAIL = os.getenv('OIDC_FALLBACK_MATCH_BY_EMAIL', '').lower() == 'true'
 
+    KEYCLOAK_AIA_ENDPOINT = f'{OIDC_OP_AUTHORIZATION_ENDPOINT}?client_id={OIDC_RP_CLIENT_ID}'
+    KEYCLOAK_API_ENDPOINT = os.getenv('KEYCLOAK_URL_API')
+    KEYCLOAK_ADMIN_CLIENT_ID = os.getenv('KEYCLOAK_ADMIN_CLIENT_ID')
+    KEYCLOAK_ADMIN_CLIENT_SECRET = os.getenv('KEYCLOAK_ADMIN_CLIENT_SECRET')
+    KEYCLOAK_ADMIN_TOKEN_ENDPOINT = os.getenv('KEYCLOAK_ADMIN_URL_TOKEN')
+else:
+    OIDC_RP_CLIENT_ID = None
+    OIDC_RP_CLIENT_SECRET = None
+    OIDC_RP_SIGN_ALGO = None
+    OIDC_OP_AUTHORIZATION_ENDPOINT = None
+    OIDC_OP_TOKEN_ENDPOINT = None
+    OIDC_OP_USER_ENDPOINT = None
+    OIDC_OP_JWKS_ENDPOINT = None
```

Comment:
Docs were throwing errors about missing oidc settings. So setting these to none.